docker-compose.yml

```diff
@@ -579,7 +579,26 @@ services:
       - s3
       - sqs
       - grapl-provision
-
+      - grapl-graphql-endpoint
+      - grapl-ux-router
+  
+  grapl-engagement-view-uploader:
```

Comment:
a new service that waits for s3 bucket to exist (courtesy of grapl-provision) and then uploads the engagement view to s3